pkg/image/validation/kubernetes.go

```diff
@@ -115,41 +115,38 @@ func validateNodes(k8s *image.Kubernetes) []FailedValidation {
 
 func validateNetwork(k8s *image.Kubernetes) []FailedValidation {
 	var failures []FailedValidation
-	ip := k8s.Network.APIVIP
 
-	numNodes := len(k8s.Nodes)
-	if numNodes <= 1 {
-		// Single node cluster, node configurations are not required
+	if k8s.Network.APIVIP == "" {
+		failures = append(failures, FailedValidation{
+			UserMessage: "The 'apiVIP' field is required in the 'network' section when defining entries under 'nodes'.",
+		})
+
 		return failures
 	}
 
-	if ip == "" {
+	parsedIP, err := netip.ParseAddr(k8s.Network.APIVIP)
+	if err != nil {
 		failures = append(failures, FailedValidation{
-			UserMessage: "The 'apiVIP' field is required in the 'network' section when defining entries under 'nodes'.",
+			UserMessage: fmt.Sprintf("Invalid address value %q for field 'apiVIP'.", k8s.Network.APIVIP),
+			Error:       err,
 		})
+
+		return failures
 	}
 
-	if ip != "" {
-		parsedIP, err := netip.ParseAddr(ip)
-		if err != nil {
-			failures = append(failures, FailedValidation{
-				UserMessage: fmt.Sprintf("Invalid APIVIP address %q for field 'apiVIP'.", ip),
-				Error:       err,
-			})
-		}
+	if !parsedIP.Is4() && !parsedIP.Is6() {
```

Comment:
These two checks are redundant, when the address is parsed successfully it's either IPv4 or IPv6. And would also not pass the following following IsGlobaUnicast().